### ISSUE TITLE
[REGRESSION] Android API not working for Android 11 after 1.22

### ISSUE THREAD

**Issue by OxNinja** [microsoft/playwright#14838]
**Context:**
- Playwright Version: `1.22.1`
- Operating System: Linux
- Node.js version: `15.14.0`
- Browser: Chromium
- Extra: N/A

**Code Snippet**

```javascript
const {_android: android} = require('playwright');

(async () => {
  const [device] = await android.devices();
  const context = await device.launchBrowser();
  // do stuff after
})();
```

**Describe the bug**

On Android 11 this code hangs at the `about:blank` page, but not on Android 10.

**Comment by dgozman** [comment]
@mxschmitt Would you like to take a look?

**Comment by mxschmitt** [comment]
Are you using an Android emulator or a real device?

Does an exception gets thrown?

Could you set the `pw:android` env var? This should give us more information whats going wrong.

**Comment by OxNinja** [comment]
Yes I am using a real device
No, there is no error or exception
With `export 'pw:android'=1` I got `export: not valid in this context: pw:android`, is there any other way to set this env var?

**Comment by mxschmitt** [comment]
ah my bad! 

```
// Linux
export DEBUG=pw:android
// PowerShell
$Env:DEBUG="pw:android"
// Batch
set DEBUG=pw:android
```

**Comment by OxNinja** [comment]
Thanks, here is what I got with the env var set:

![image](https://user-images.githubusercontent.com/57400445/173563200-f3be01fb-0c02-476b-980b-1931b87fb01d.png)

With the promise not resolving

**Comment by mxschmitt** [comment]
I tried to run it with an emulator on android 11 which is SDK 30 which was working fine. So most likely something with the device which is off.

Can you verify that you followed the android requirements? https://playwright.dev/docs/api/class-android

**Comment by mxschmitt** [comment]
Good to know! It might be caused by https://github.com/microsoft/playwright/pull/13248.

Could you try setting the `DEBUG=pw:api,pw:android,pw:adb*` env var?

**Comment by OxNinja** [comment]
OK so after hours of debugging, I don't know why I got this error, but now it is gone.
I installed PW with `npm -i playwright@version`, with `version` one of `latest,1.22.1,1.21.0` and tested with my provided script above in this post.
All version worked and started Chrome on the device.

Maybe since I rebooted 10+ times and had some kernel upgrade, including npm update... The problem I had back then was fixed by black magic. On one hand I am frustrated to not being able to understand why it did not worked, on the other hand I am glad I can now continue to dev.

Thanks for your time :smile:
